src/main/java/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -269,11 +269,10 @@ private void initializeAttributes() {
         R.attr.conversation_item_incoming_bubble_color,
         R.attr.conversation_item_outgoing_bubble_color,
     };
-    final TypedArray attrs      = context.obtainStyledAttributes(attributes);
-
-    incomingBubbleColor = attrs.getColor(0, Color.WHITE);
-    outgoingBubbleColor = attrs.getColor(1, Color.WHITE);
-    attrs.recycle();
```

Comment:
where is `recycle()` gone?

that seems at least more than what the linter is telling me :)

Reply:
The try block handles automatic closing of resources without needing to remember such manual calling to recycle with potential forgetting to do so, that is what the warning was about, calling recycle inside the try would cause a second attempt to recycle an already recycled resource results in an exception